### PR TITLE
VCDGear, new recipe

### PR DESCRIPTION
**Pull request by Begasus**
Just a recipe, haven't checked if it actually works with a VCD (don't have one at hand) and not checked on x86_64 (yet)

### CODE REVIEW

File: haiku-apps/vcdgear/vcdgear-1.1.recipe [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+BUILD()
+{
+	true
+}
+
+INSTALL()
+{
+	mkdir -p $binDir $docDir
+	cp -f changes $docDir
+	cp -f readme $docDir
```
Comment (miqlas) [inline haiku-apps/vcdgear/vcdgear-1.1.recipe]: Please, use docdir/vcdgear here and at L45.

File: haiku-apps/vcdgear/vcdgear-1.1.recipe [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	chmod +x VCDGear
+	cp -f VCDGear $binDir
```
Comment (miqlas) [inline haiku-apps/vcdgear/vcdgear-1.1.recipe]: Can you swap the 2 last line? You have to adjust the path for chmod!

File: haiku-apps/vcdgear/vcdgear-1.1.recipe [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+HOMEPAGE="http://www.vcdgear.com"
+SOURCE_URI="http://www.vcdgear.com/files/VCDGear_1.1_beos5.zip"
+CHECKSUM_SHA256="21e91fc079e1a7db4c453dd72f92d978eb444f5fc7d09f091bb9834dc8c6c778"
+LICENSE="MIT"
+COPYRIGHT="2000 Dracore
+	Be-NiB"
+REVISION="1"
+SOURCE_DIR="VCDGear"
+
+ARCHITECTURES="any"
```
Comment (korli) [inline haiku-apps/vcdgear/vcdgear-1.1.recipe]: This should be x86_gcc2 only, this is an ELF binary.

**Comment by Begasus** [comment]
@miqlas this is a binary, not a script :)

**Comment by Begasus** [comment]
moved it to the correct place "media-video"

File: haiku-apps/vcdgear/vcdgear-1.1.recipe (hunk 1 above)
Comment (Begasus) [reply haiku-apps/vcdgear/vcdgear-1.1.recipe]: it's creating docdir/vcdgear already ;)

**Comment by miqlas** [comment]
Wouldn't it be better to re-port this stuff?

File: haiku-apps/vcdgear/vcdgear-1.1.recipe (hunk 2 above)
Comment (Begasus) [reply haiku-apps/vcdgear/vcdgear-1.1.recipe]: swapped (thanks)

**Comment by Begasus** [comment]
There is no source for it afaik (as in the readme file "this is freeware, not open source")

**Comment by miqlas** [comment]
We can ask for it…

**Comment by korli** [comment]
I don't understand why there are two recipes in the PR.

**Comment by Begasus** [comment]
Ah I see, one should be 1.1 the other one should be 1.0.1 (iirc) (not in Haiku atm)
the older one still holds the binary for vcd_fs (from BeOS R4)

**Comment by scottmc** [comment]
What's left to do here?  Is it ready for merge?

**Comment by Begasus** [comment]
Should be ok to merge, ask @humdingerb :)

**Comment by humdingerb** [comment]
Why me? :)
I don't get why there are two recipes in this PR either. Can't the 1.0.1 be dropped?
Otherwise, I find the SUMMARY and DESCRIPTION very awkward...

**Comment by Begasus** [comment]
@humdingerb ... you were the one requesting this one ;)

**Comment by Begasus** [comment]
Last changes uses the binary from 1.1 and the add-on from 1.0.1 ...

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**